src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateLegalEntityCommandTests/WhenTheUserIsNotWhitelistedForExpressionOfInterest.cs

```diff
@@ -15,7 +14,7 @@ public override void Arrange()
         {
             base.Arrange();
 
-            AuthorizationService.Setup(x => x.IsAuthorized(FeatureType.ExpressionOfInterest)).Returns(false);
+            AuthorizationService.Setup(x => x.IsAuthorized("Feature.ExpressionOfInterest")).Returns(false);
```

Comment:
should this be EmployerFeature. like the other one?

Reply:
It should

Reply:
Updated in next commit.